geotrek/sensitivity/fixtures/rules.json

```diff
@@ -0,0 +1,597 @@
+[
+    {
+        "pk": 1,
+        "model": "sensitivity.rule",
+        "fields": {
+            "code": "DOGS-FORBIDDEN",
+            "name": "Chiens interdits, m\u00eame tenus en laisse",
+            "name_en": "Dogs forbidden, even on a leash",
+            "pictogram": "upload/DOGS-FORBIDDEN_011.svg",
```

Comment:
I think all fixture files should be prefixed, by default, in upload/sensitivity/rules/ folder

```
upload/sensitivity/rules/DOGS-FORBIDDEN_011.svg
```